pkgs/development/libraries/gsettings-qt/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchbzr, pkgconfig, qmake, glib, qttools, qtdeclarative, gobjectIntrospection }:
+
+stdenv.mkDerivation rec {
+  name = "gsettings-qt-${version}";
+  version = "0.1.20170824";
+
+  src = fetchbzr {
+    url = http://bazaar.launchpad.net/~system-settings-touch/gsettings-qt/trunk;
+    rev = "85";
+    sha256 = "1hf3jj6bygz20xhprvbpv2csv0xvw0al0h3w0kpgbfqdadqhb30r";
+  };
+
+  nativeBuildInputs = [
+    pkgconfig
+    qmake
+    gobjectIntrospection
+  ];
+
+  buildInputs = [
+    glib
+    qttools
+    qtdeclarative
+  ];
+
+  # Parallel build not supported. It causes error when linking.
+  enableParallelBuilding = false;
+  
+  doCheck = false;
+
+  patchPhase = ''
+    substituteInPlace GSettings/gsettings-qt.pro --replace '$$[QT_INSTALL_QML]' "$out/$qtQmlPrefix"
+    substituteInPlace GSettings/gsettings-qt.pro --replace '$$[QT_INSTALL_BINS]' "${qtdeclarative.dev}/bin"
```

Comment:
`$$[QT_INSTALL_BINS]` needs to be replaced with `$out/bin` which will be writable.

Reply:
In this case `$$[QT_INSTALL_BINS]` is being used to locate the binary `qmlplugindump`, which can be found as `${qtdeclarative.dev}/bin` or maybe in `$PATH`.

```
qmltypes.extra = export LD_PRELOAD=../src/libgsettings-qt.so.1; $$[QT_INSTALL_BINS]/qmlplugindump -notrelocatable GSettings 1.0 .. > $(INSTALL_ROOT)/$$installPath/plugins.qmltypes
```